nf_core/modules/remove.py

```diff
@@ -58,5 +58,27 @@ def remove(self, module):
 
         log.info("Removing {}".format(module))
 
+        # Remove entry from modules.json
+        if not self.remove_modules_json_entry(module):
+            return False
```

Comment:
Similar to the other command, I would rather print that the module entry couldn't be found in the `modules.json` and not prompt for it.